pkg/asset/manifests/content/bootkube/kube-system-configmap-etcd-serving-ca.go

```diff
@@ -0,0 +1,21 @@
+package bootkube
+
+const (
+	// KubeSystemConfigmapEtcdServingCA  is the constant to represent contents of kube-system-configmap-etcd-serving-ca.yaml file
+	KubeSystemConfigmapEtcdServingCA = `
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: etcd-serving-ca
+  namespace: kube-system
+  labels:
+    tier: "control-plane"
+    k8s-app: "kube-apiserver"
+    openshift.io/control-plane: "true"
+    openshift.io/component: "api"
+data:
+  ca-bundle.crt: |
+    {{ .Assets | load .EtcdServingCA | indent 4 }}
```

Comment:
Please link me to where this gets substituted.  I would expect this to fail.

Reply:
Fixed